_drafts/assert.md

```diff
@@ -0,0 +1,605 @@
+---
+title: "Using Asserts in Embedded Systems"
+description:
+  "Asserts are a quick and proven way to prevent bugs in embedded systems. An
```

Comment:
MAYBE:
> An optimal assert implementation both maximizes the context around the assert while minimizing code size and RAM usage.

->

An optimal assert implementation maximizes the context around the assert with minimal code and RAM overhead.